libres/tests/analysis/test_save_parameters.cpp

```diff
@@ -0,0 +1,87 @@
+#include <filesystem>
+#include <iostream>
+#include <fstream>
+
+#include "catch2/catch.hpp"
+
+#include <ert/enkf/enkf_fs.hpp>
+#include <ert/analysis/update.hpp>
+#include <ert/enkf/ensemble_config.hpp>
+#include <ert/enkf/enkf_config_node.hpp>
+#include <ert/enkf/enkf_node.hpp>
+#include <ert/enkf/enkf_defaults.hpp>
+#include <ert/enkf/enkf_fs.hpp>
+#include <ert/enkf/local_ministep.hpp>
+#include <ert/util/type_vector_functions.hpp>
+
+#include "../tmpdir.hpp"
+
+extern "C" void enkf_fs_umount(enkf_fs_type *fs);
+
+TEST_CASE("Write and read a matrix to enkf_fs instance", "[analysis]") {
```

Comment:
maybe add the label `[private]` to show that this test private code

Reply:
if you are going for the BDD style of testing then you should use the proper macros. https://github.com/catchorg/Catch2/blob/devel/docs/test-cases-and-sections.md#bdd-style-test-cases